WORKSPACE.bazel

```diff
@@ -0,0 +1,44 @@
+workspace(name = "com_github_goki_vulkan")
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "io_bazel_rules_go",
+    sha256 = "dd926a88a564a9246713a9c00b35315f54cbd46b31a26d5d8fb264c07045f05d",
+    urls = [
+        "https://mirror.bazel.build/github.com/bazelbuild/rules_go/releases/download/v0.38.1/rules_go-v0.38.1.zip",
+        "https://github.com/bazelbuild/rules_go/releases/download/v0.38.1/rules_go-v0.38.1.zip",
+    ],
+)
+
+load("@io_bazel_rules_go//go:deps.bzl", "go_register_toolchains", "go_rules_dependencies")
+
+go_rules_dependencies()
+
+go_register_toolchains(version = "1.18.10")
+
+vulkan_mac_deps_version = "1.3.239.0"
+
+http_archive(
+    name = "com_github_goki_vulkan_mac_deps",
+    sha256 = "348bc84c0fc1f1e79fb28bcf83454faa2a84c4d4c2286225d4413830a3c0a29c",
+    strip_prefix = "vulkan_mac_deps-%s" % vulkan_mac_deps_version,
+    url = "https://github.com/goki/vulkan_mac_deps/archive/refs/tags/%s.tar.gz" % vulkan_mac_deps_version,
```

Comment:
I mean this one fore instance, would like to contain all base vulkan bindings and deps within this organisation.

Reply:
@xlab thanks -- @garymm is the one who manages all the Bazel stuff, so if it is Ok with you, I can add him and he can do that?

What are the current prospects for your availability to dig into this stuff again and get a cleaner version of it working here?  i.e., not using the old version of c-for-go, and automating the logic so the manual patching isn't required?  I don't have time to dig into this level of issues at this point, and my hacky solution is working OK for now..

Reply:
@rcoreilly yes, feel free to add your contributors also as members ;)

> i.e., not using the old version of c-for-go, and automating the logic so the manual patching isn't required? I don't have time to dig into this level of issues at this point, and my hacky solution is working OK for now..

Yes  I still have hopes to dig into this, for this reason I don't want to just blindly merge the PR yet. Although I appreciate the fact that you and @garymm keep this alive and fresh, so I just want it to be available to everyone, even it's not "officially" cleaned up and merged.

For that reason I just ask about publishing build deps / artefacts under this org, so even if your personal focus changes, the repos are not gone and still usable by enthusiasts. I seen so many good repos on GH gone...
So yeah, this was my main motivation to add you into `vulkan-go` org, so all necessities could be stored there.

But in long term, I still want to clean this up. ✌️

Reply:
I added vulkan-go/vulkan_mac_deps.
Just update the references to goki/vulkan_mac_deps and it should work.

https://github.com/vulkan-go/vulkan_mac_deps